### DISCUSSION TITLE
My github hacked and they changed emails and github name

### DISCUSSION THREAD

**Discussion by milosDeljanin53** [community/community#76220] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Somebody changed my GitHub password and they removed all my emails.
But it's strange how it is possible?
Because I've enabled 2-factor authentication with authenticator extension, and they can't know my Gmail password.
And also he changed my GitHub's origin name, so I can't find my GitHub account.
I don't know he removed or changed the name of the account.
Can support team back my account of primary email  and recover all repos to me?

**Comment by ghost** [comment]
Hello @TopDevBeast53 

It all depends what has happened. I suggest you to contact github support as soon as possible and try to get back your account. if your repos where deleted probably you will not be able to recover them, this applies to projects as well since once you delete repo/project from github generally its totaly lost.

However its worth to try to at least get your account back and then try to recover your repositories from your local machine (create new repositories on github and push local clones).

hope this helps.

**Reply by milosDeljanin53** [reply]
Thanks for reply.
Can I recover my account with my repos?

**Reply by ghost** [reply]
As i said it is very specific case and no one can say that you can or can't. 

If you can't logg in github on your own you should contact [Github Support Team](https://support.github.com/). explain your problem and wait for their answers. if you manage to recover your account congrats, but there starts your next problem - repos. if repos are deleted you will not be able to recover them. in this case only thing you can do is to create new repositories with same names and then push code from your local machine if you have clonned them before, if not then you will not be able to get your repos back.

**Comment by Munashe-Njanji** [comment]
> ### Select Topic Area
> General
> 
> ### Body
> Somebody changed my GitHub password and they removed all my emails. But it's strange how it is possible? Because I've enabled 2-factor authentication with authenticator extension, and they can't know my Gmail password. And also he changed my GitHub's origin name, so I can't find my GitHub account. I don't know he removed or changed the name of the account. Can support team back my account of primary email and recover all repos to me?

This just occured to me as well

**Comment by msi-end** [comment]
I'm sorry to hear about your situation. Losing access to your GitHub account can be distressing. However, there are steps you can take to attempt to recover your account:

1. **Contact GitHub Support:** Even if you can't access your account directly, you can reach out to GitHub support via their [support page](https://support.github.com/contact) and explain your situation. Provide as much detail as possible, including your account's previous email addresses, usernames, and any other relevant information. They may be able to assist you in recovering your account.

2. **Check for Account Recovery Options:** GitHub provides several options for account recovery, such as using your recovery codes if you set them up previously, or verifying your identity through other means. Check if any of these options are available to you.

3. **Check for Suspicious Activity:** If you suspect that your account has been compromised, it's essential to take steps to secure your other accounts as well. Change passwords for any accounts that share the same or similar credentials, and consider enabling additional security measures such as two-factor authentication where possible.

4. **Stay Vigilant:** Keep an eye on your other online accounts for any signs of unusual activity. It's possible that the individual who compromised your GitHub account may attempt to access other accounts as well.
